.gitattributes

```diff
@@ -1,5 +1,5 @@
-# Prevent Git to auto detect text files and perform LF normalization.
-* -text
+# Set default behavior to automatically normalize line endings.
+* text=auto
```

Comment:
`text=auto` will cause `git checkout` to produce CRLF endings on Windows.

Whereas the other change in this PR causes `rush init` to write LF line endings.

This seems inconsistent -- is our policy "native line endings for each OS" or "consistently use LF across every OS"?

In https://github.com/microsoft/rushstack/issues/1467#issuecomment-522682958 you seemed to be advocating the latter.

Reply:
Updated to write OS-default line endings.